journal/modernized/07-chapter-6.adoc

```diff
@@ -282,7 +282,7 @@ We this evening took a walk, to see an elderly man,
 who was a member of the Church of England, but embraced the principles of Friends,
 from conviction, on reading some of their writings:
 he resides alone in a retired situation, about a mile in the country, has a garden,
-and with what it produces, etc., is worth about £14 per year:
+and with what it produces, etc., is worth about Â£14 per year:
```

Comment:
Hmmm.. do you see that weird character?  Did you add that? Or maybe I wonder if it's some sort of encoding artifact that got introduced because of `jones`.